### PR TITLE
Ft 300

### PR DESCRIPTION
**Pull request by UltronDestroyer**
This adds the FT300, makes a prefix change fix, and moves forward the missing packages.

### CODE REVIEW

File: robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <xacro:macro name="robotiq_fts150" params="parent prefix *origin">
 
         <!-- mount the fts to the robot -->
-        <joint name="{prefix}fts_fix" type="fixed" >
+        <joint name="${prefix}fts_fix" type="fixed" >
```
Comment (UltronDestroyer) [inline robotiq_force_torque_sensor/urdf/robotiq_fts150.urdf.xacro:7]: This commit marks the missing syntax fix.

File: robotiq_force_torque_sensor/urdf/robotiq_fts300.urdf.xacro [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```
Comment (UltronDestroyer) [inline robotiq_force_torque_sensor/urdf/robotiq_fts300.urdf.xacro:1]: This commit adds the FT300 urdf.xacro to the robotiq package models

File: robotiq_s_model_articulated_gazebo/CMakeLists.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 2.8.3)
```
Comment (UltronDestroyer) [inline robotiq_s_model_articulated_gazebo/CMakeLists.txt]: This commit moves the missing packages forward

File: robotiq_force_torque_sensor/nodes/rq_sensor.cpp.save [hunk 4]
```diff
@@ -0,0 +1,220 @@
+/* Software License Agreement (BSD License)
```
Comment (shaun-edwards) [inline robotiq_force_torque_sensor/nodes/rq_sensor.cpp.save:1]: Did you mean to commit this file?

**Comment by UltronDestroyer** [comment]
I'm closing this and making it cleaner/fixing it up.

**Comment by wxmerkt** [comment]
Any update on a cleaner PR adding the FTS300? We're having trouble using the dual_ur5_husky simulation because of loads of missing packages and old dependencies (e.g. the groovy naming of robotiq_s_model_vis)

**Comment by Danfoa** [comment]
@wxmerkt URDF model is on #138. The driver is fully functional at the moment, only the URDF model was missing.

**Pull request closed without merging**